bandit.yml

```diff
@@ -0,0 +1,5 @@
+# B322: checks for `input()` which is unsafe in Python 2, but safe in Python 3.
+# B104: checks for binding 0.0.0.0 interface, which should be fine for containers.
+# B301: checks for pickle usage, which is a necessary evil.
+# B303: checks for insecure hash functions like md5, which is good enough for current use cases.
```

Comment:
@wochinge @joejuzl I'm having a doubt about this one. I think that the current use we make of sha1/md5 is fine today, but we cannot guarantee that the future uses are fine as well. It might make sense to not skip it, and locally disable the checks by adding `# nosec` comments. What do you think?

Reply:
Yeah I agree - safer to skip it on a case-by-case basis where we know it is not being used for security purposes.